product_pricelist_supplierinfo/models/product_template.py

```diff
@@ -37,6 +37,12 @@ def _get_supplierinfo_pricelist_price(
             # Verbatim copy of part of product.pricelist._compute_price_rule.
             qty_uom_id = self._context.get('uom') or self.uom_id.id
             price_uom = self.env['uom.uom'].browse([qty_uom_id])
+
```

Comment:
Please don't add empty lines inside a method. It's discouraged in PEP8.

Reply:
If it is a PEP and the OCA likes to follow it you should make a issue in pylint-odoo in order to enable it

if not, maybe OCA doesn't like to follow this particular PEP

So, you can avoid checking this again and again for each PR and delegate it to the CI

Reply:
Well, as I'm the reviewer and original coder of this module, don't want to introduce these ugly lines in the module, so please remove them, independently from the pylint thing.

Reply:
Well, as we are the creator and contributor of this change of this module and it is under umbrella guidelines and CI we don't want fix Pedro's guidelines that are not discussed in the OCA and the CI PEP style is green

Reply:
OK, as you prefer, then don't ping me again on any PR.